src/actions/indexActions.js

```diff
@@ -0,0 +1,23 @@
+export function setPage(page) {
+  return {
+    type: 'SET_PAGE',
```

Comment:
Should we keep a separate CONSTANTS file like in the app? It lets us more easily configure constants in the future if there's ones that need renaming

Reply:
In what case would having a variable called `FAILED` help us over a string called `LOGIN_FAILED`?
I supposed it helps the IDE?

Reply:
In the app? I think we could put constants together so it's like `LOGIN.FAILED` and `LOGIN.SUCCESS` or something, I think that was the original point of it